src/components/frame/Extensions/pages/ExtensionDetailsPage/ExtensionDetailsPage.tsx

```diff
@@ -25,46 +25,53 @@ const ExtensionDetailsPage: FC = () => {
   const { extensionData } = useExtensionData(extensionId ?? '');
   const { formatMessage } = useIntl();
 
-  if (!colony) {
+  if (!colony || !extensionData) {
     return null;
   }
 
   if (!extensionData) {
-    return (
-      <div>
-        <p>{formatMessage({ id: 'extensionDetailsPage.unsupportedExtension' })}</p>
-      </div>
-    );
+    return <p>{formatMessage({ id: 'extensionDetailsPage.unsupportedExtension' })}</p>;
+  }
+
+  if (!isInstalledExtensionData(extensionData)) {
```

Comment:
Why are you returning null when extension is not installed?

Reply:
Removed